### PR TITLE
feat(paddle backend): Manual dtype casting removal paddle --round 3

### PR DESCRIPTION
**Pull request by Madjid-CH**

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy/functional/backends/paddle/general.py [hunk 1]
```diff
@@ -45,24 +47,17 @@ def _check_query(query):
     )
 
 
+@with_unsupported_dtypes(
+    {"2.5.1 and below": ("float16", "int16", "int8")}, backend_version
```
Comment (vedpatwardhan) [inline ivy/functional/backends/paddle/general.py:51]: have you tested `bfloat16`?

**Review by vedpatwardhan — CHANGES_REQUESTED** [review]
Just a minor point, thanks @Madjid-CH 😄

File: ivy/functional/backends/paddle/general.py (hunk 1 above)
Comment (Madjid-CH) [reply ivy/functional/backends/paddle/general.py:51]: yes:
```python
x = paddle.to_tensor([1, 2, 3], dtype=paddle.bfloat16)
get_item(x, query=0)
Out[7]: 
Tensor(shape=[1], dtype=bfloat16, place=Place(cpu), stop_gradient=True,
       [1.])
```

Comment (vedpatwardhan) [reply ivy/functional/backends/paddle/general.py:51]: awesome, thanks!

**Review by vedpatwardhan — APPROVED** [review]
lgtm! Feel free to merge, thanks @Madjid-CH 😄
(Given that the tests aren't running in the CI due to a docker issue, I'd suggest running the tests for these functions locally and ensuring that they're passing before you merge)

**Comment by Madjid-CH** [comment]
> lgtm! Feel free to merge, thanks @Madjid-CH 😄 (Given that the tests aren't running in the CI due to a docker issue, I'd suggest running the tests for these functions locally and ensuring that they're passing before you merge)

understood

**Pull request merged**